nav2_map_server/src/map_io.cpp

```diff
@@ -261,12 +261,25 @@ void loadMapFromFile(
       normalized = (1.0f - normalized.array()).matrix();
     }
 
+    double occupied_thresh = load_parameters.occupied_thresh;
```

Comment:
Remove and use `load_parameters.occupied_thresh` directly please - we do so lots of places in the code below so changing just the one instance below this isn't consistent